src/main/java/com/backend/webproject/controller/AdminSideController.java

```diff
@@ -22,107 +23,53 @@
 public class AdminSideController {
 
     @Autowired
-    CouponsDAO couponsDAO;
-
-    @Autowired
-    EventsDAO eventsDAO;
+    AdminSideManager adminSideManager;
 
     @RequestMapping("/")
     public String showAdminPage() {
-        return "admin";
+
+        return adminSideManager.showAdminPage();
```

Comment:
You don't need to do this in the manager, you can have the name of the jsp file in the controller.

Reply:
Correction done.